lib/factories/ppt/slides.js

```diff
@@ -211,10 +210,32 @@ class SlideFactory {
         // now add the nodes which turn a shape block into a text block
         newTextBlock['p:nvSpPr'][0]['p:cNvSpPr'] = [{ $: { txBox: '1' } }];
         newTextBlock['p:spPr'][0]['a:noFill'] = [{}];
-        newTextBlock['p:txBody'][0]['a:bodyPr'] = [{ $: { wrap: 'square', rtlCol: '0' } }];
+        newTextBlock['p:txBody'][0]['a:bodyPr'] = [{ $: { rtlCol: '0' } }];
+
+        PptFactoryHelper.addParagraphPropertiesToBlock(newTextBlock['p:txBody'][0]['a:p'][0], options);
+
         newTextBlock['p:txBody'][0]['a:p'][0]['a:r'] = [{ 'a:rPr': [{ $: { lang: 'en-US', smtClean: '0' } }], 'a:t': text }];
         newTextBlock['p:txBody'][0]['a:bodyPr'][0]['a:spAutoFit'] = [{}];
-        newTextBlock['p:txBody'][0]['a:p'][0]['a:endParaRPr'] = [{ $: { lang: 'en-US' } }]; // this MUST go last!
+
+        let textRunPropertyBlock = newTextBlock['p:txBody'][0]['a:p'][0]['a:r'][0]['a:rPr'][0];
+        let textBodyPropertyBlock = newTextBlock['p:txBody'][0]['a:bodyPr'][0];
+        let endParagraphRun = [{ $: { lang: 'en-US' } }];
+
+        PptFactoryHelper.addLinePropertiesToBlock(newTextBlock['p:spPr'][0], options.line);
+
+        if (options.textColor !== undefined) {
+            textRunPropertyBlock['a:solidFill'] = PptFactoryHelper.createColorBlock(options.textColor);
+        }
+
+        if (options.fontFace) {
```

Comment:
you check for undefined above and then use type coercion here. I'd rather be consistent and do the strict type checking.

Reply:
Done.